src/components/home/Hero.vue

```diff
@@ -18,10 +18,14 @@
         <transition name="slide-up" type="animation" appear>
           <h1 class="title is-1" id="hero-text">
             A module registry for
-            <span id="deno-text">Deno</span>, <br />
+            <!-- <span id="deno-text">Deno</span>, <br /> -->
+            <a class="gradient-text" href="https://deno.land/">
```

Comment:
```suggestion
            <a class="gradient-text" href="https://deno.land/">
```
Personally, I'm moreso in favor of the design before here. I am biased though, as I created the original.

Reply:
I'd also go for the old design because of the text being over the footprints in the background and messing up the contrast. It's very minor problem but still.

Reply:
Okay, so I think that for now, we should revert to the old design. I agree with Max - we need appropriate contrast.